src/pages/sighting/featuredPhoto/FeaturedPhotoSelector.jsx

```diff
@@ -9,21 +8,32 @@ import CustomAlert from '../../../components/Alert';
 import usePatchSighting from '../../../models/sighting/usePatchSighting';
 import Button from '../../../components/Button';
 import StandardDialog from '../../../components/StandardDialog';
+import usePatchIndividual from '../../../models/individual/usePatchIndividual';
 
 export default function FeaturedPhotoSelector({
   open,
   onClose,
   assets,
-  sightingId,
+  sightingId = null,
+  individualId = null,
   currentFeaturedPhotoId,
-  refreshSightingData,
+  refreshData,
 }) {
   const theme = useTheme();
   const [selectedPhoto, setSelectedPhoto] = useState(
     currentFeaturedPhotoId,
   );
 
-  const { updateProperties, loading, error } = usePatchSighting();
+  const {
+    updateProperties,
+    loading: sightingLoading,
+    error: sightingError,
+  } = usePatchSighting();
+  const {
+    updateIndividualProperties,
+    loading: individualLoading,
+    error: individualError,
+  } = usePatchIndividual();
```

Comment:
Another option would be to pass the mutation in as a prop. This way we'll have to continue adding `usePatchXYZ...` to this file whenever the component gets used for another entity type

Reply:
Good idea!

Reply:
Respectfully, gonna punt this until we hit the scenario where we're gonna have to use this in another entity type and cross that bridge when we get there.